### PR TITLE
Removido temporariamente menu de acesso à doc em inglês do Ecommerce e ajustes de frases

### PR DESCRIPTION
**Pull request by andrebparpaiola**

### CODE REVIEW

**Comment by andrebparpaiola** [comment]
@bcunhapicpay

File: static/swagger/ecommerce-en.json [hunk 1]
```diff
@@ -9,33 +9,33 @@
...
   },
   "tags": [
     {
-      "name": "Criar pedido para pagamento"
+      "name": "Create order for payment"
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:16]: Create an order for a payment

File: static/swagger/ecommerce-en.json [hunk 2]
```diff
@@ -9,33 +9,33 @@
...
     },
     {
-      "name": "Capturar pagamento"
+      "name": "Capture payment"
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:19]: Payment Capture

File: static/swagger/ecommerce-en.json [hunk 3]
```diff
@@ -9,33 +9,33 @@
...
     },
     {
-      "name": "Cancelar pedido de pagamento"
+      "name": "Cancel payment request"
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:22]: Payment Cancel request

File: static/swagger/ecommerce-en.json [hunk 4]
```diff
@@ -9,33 +9,33 @@
...
     },
     {
-      "name": "Consultar Status do pedido"
+      "name": "Check Order Status"
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:25]: Order Check Status

File: static/swagger/ecommerce-en.json [hunk 5]
```diff
@@ -9,33 +9,33 @@
...
-      "name": "Notificação de mudança no pedido"
+      "name": "Order change notification"
     }
   ],
   "paths": {
     "/payments": {
       "post": {
         "tags": [
-          "Criar pedido para pagamento"
+          "Create order for payment"
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:35]: Creante an order for a payment

File: static/swagger/ecommerce-en.json [hunk 6]
```diff
@@ -9,33 +9,33 @@
...
+        "summary": "Request to Create order for payment",
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:38]: Create an order for payment Request

File: static/swagger/ecommerce-en.json [hunk 7]
```diff
@@ -77,10 +77,10 @@
     "/payments/{referenceId}/capture": {
       "post": {
         "tags": [
-          "Capturar pagamento"
+          "capture payment"
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:80]: Payment Capture

File: static/swagger/ecommerce-en.json [hunk 8]
```diff
@@ -77,10 +77,10 @@
...
+        "description": "\nThe partial or total payment capture is realized out under the condition that the payment request is \ncreated with the `autoCapture` field set to `false` and the consumer has made the payment. \nThe request will be awaited to effect the charge, with the correct amount, informed in the `amount` field. \nThe value can be the same or less than the authorized value.Captures with a value greater than the authorized value will be \nrejected.\n\nThe return of the request informs the updated values ​​pertinent to the transaction, located in the\n`summary` field:\n- **`authorized` field:** contains the authorized value; \n- **Field `paid`:** registers the amount paid (captured).\n\n\n**Information:**\n- The capture request is only allowed when the transaction is authorized. In the situation where \nthe capture request does not occur within 5 consecutive days, counting from the authorization, the order is canceled \automatically;\n\n- After the successful capture, even if of partial value, it will not be possible request\nnew capture of the same order/transaction.",
+        "summary": "Request to capture payment total or partial",
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:83]: Payment Capture Request (total or partial)

File: static/swagger/ecommerce-en.json [hunk 9]
```diff
@@ -134,10 +134,10 @@
     "/payments/{referenceId}/refunds": {
       "post": {
         "tags": [
-          "Cancelar pedido de pagamento"
+          "Cancel payment request"
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:137]: Payment Cancel Request

File: static/swagger/ecommerce-en.json [hunk 10]
```diff
@@ -134,10 +134,10 @@
...
+        "summary": "Request to cancel / partial or full refund",
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:140]: Cancel Request / Partial or Full Refund

File: static/swagger/ecommerce-en.json [hunk 11]
```diff
@@ -209,10 +209,10 @@
     "/payments/{referenceId}/status": {
       "get": {
         "tags": [
-          "Status do pedido"
+          "Check Order Status"
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:212]: Order Check Status

File: static/swagger/ecommerce-en.json [hunk 12]
```diff
@@ -209,10 +209,10 @@
...
         ],
         "description": "Allows to consult and identify the current status of the payment request, through the `status` field. \nThe return of the request informs the updated values ​​pertaining to the payment request, located \nin the `summary` field:\n- **Field `authorized`:** authorized value;\n- **Field `paid`:** amount paid (captured);\n- **Field `refunded`:** amount canceled.",
-        "summary": "Requisição de detalhes sobre o pedido",
+        "summary": "Request for order details",
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:215]: Order details Request

File: static/swagger/ecommerce-en.json [hunk 13]
```diff
@@ -257,13 +257,13 @@
...
-        "summary": "Notificação para informar atualização do pedido",
+        "description": "\nThrough the callback, the merchant's system is notified when there is a change of state \n(status) in the order.\nThe message is sent to the address identified when creating the order, through the field\n`callbackUrl`.This way, it is possible to know when something has changed and query the order again to \nupdate the records.\n\nAs a security measure, updates are not communicated directly to the merchant\n\ncallbackUrl`s system.This is as a guarantee that only the retailer has access \nto the data of their orders and that they do not come to the knowledge of third parties, especially malicious people.\n\nThe merchant's system must always return a successful response (status http 200) for this call.\n\nIn case the trader's system is down, there will be another 21 callback attempts.",
+        "summary": "Notification to inform order update",
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:266]: Order update notification

File: static/swagger/ecommerce-en.json [hunk 14]
```diff
@@ -583,7 +583,7 @@
         ]
       },
       "PaymentResponse200": {
-        "title": "Resposta à requisição de pagamentos",
+        "title": "Response to payment request",
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:586]: Payment Request response body

File: static/swagger/ecommerce-en.json [hunk 15]
```diff
@@ -628,7 +628,7 @@
         ]
       },
       "CapturePaymentRequest": {
-        "title": "Requisição para captura de pagamento",
+        "title": "Request for payment capture",
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:631]: Payment capture request

File: static/swagger/ecommerce-en.json [hunk 16]
```diff
@@ -670,7 +670,7 @@
             "example": "paid"
           },
           "summary": {
-            "title": "Resumo dos valores do pagamento",
+            "title": "Summary of payment amounts",
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:673]: Payment amounts summary

File: static/swagger/ecommerce-en.json [hunk 17]
```diff
@@ -736,7 +736,7 @@
             "type": "string"
           },
           "summary": {
-            "title": "Resumo dos valores do pagamento",
+            "title": "Summary of payment amounts",
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:739]: Payment amounts summary

File: static/swagger/ecommerce-en.json [hunk 18]
```diff
@@ -793,7 +793,7 @@
             "example": "authorized"
           },
           "summary": {
-            "title": "Resumo dos valores do pagamento",
+            "title": "Summary of payment amounts",
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:796]: Payments amounts summary

File: static/swagger/ecommerce-en.json [hunk 19]
```diff
@@ -842,18 +842,18 @@
         ]
       },
       "CallbackResponse200": {
-        "title": "Resposta da requisição de callback",
+        "title": "Callback request response",
         "type": "string",
-        "example": "Vamos apenas validar o status HTTP 200."
+        "example": "Let's just validate the status HTTP 200."
```
Comment (bcunhapicpay) [inline static/swagger/ecommerce-en.json:847]: We gonna just validate the HTTP 200 status

File: static/swagger/picpay-1-click-en.json [hunk 20]
```diff
@@ -16,7 +16,7 @@
   "paths": {
     "/payments/charge": {
       "post": {
-        "summary": "Criar cobrança",
+        "summary": "Create billing",
```
Comment (bcunhapicpay) [inline static/swagger/picpay-1-click-en.json:19]: Billing Create

File: static/swagger/picpay-1-click-en.json [hunk 21]
```diff
@@ -123,14 +123,16 @@
...
-          "Pagamentos"
+          "Payments"
         ]
       }
     },
     "/payments/{transaction_id}/refund": {
       "post": {
-        "summary": "Realizar um estorno total ou parcial",
-        "description": "This resource allows the seller to make a total or partial reversal of a payment. \nIf you specify a refund amount lower than was charged, the original transaction will be canceled and we will make a second charge to the user's wallet with the difference amount.",
+        "summary": "Perform a full or partial reversal",
```
Comment (bcunhapicpay) [inline static/swagger/picpay-1-click-en.json:132]: reversal or refund?

File: static/swagger/picpay-1-click-en.json [hunk 22]
```diff
@@ -321,7 +323,7 @@
                     }
                   },
                   "badTokenFormat": {
-"description": "Response when resource is called with token that invalid JWT.",
```
Comment (bcunhapicpay) [inline static/swagger/picpay-1-click-en.json:324]: Response when resource is called with a invalid JWT token.

File: static/swagger/picpay-1-click-en.json [hunk 23]
```diff
@@ -510,10 +512,10 @@
     },
     "schemas": {
       "ChargeRequestBody": {
-        "title": "Criar cobrança",
+        "title": "Create billing",
```
Comment (bcunhapicpay) [inline static/swagger/picpay-1-click-en.json:515]: Billing Create

File: static/swagger/picpay-1-click-en.json [hunk 24]
```diff
@@ -547,7 +549,7 @@
         }
       },
       "RefundResponseBody": {
-        "title": "Reembolso realizado parcialmente",
+        "title": "Partially paid refund",
```
Comment (bcunhapicpay) [inline static/swagger/picpay-1-click-en.json:552]: Partially Refunded

File: static/swagger/picpay-1-click-en.json [hunk 25]
```diff
@@ -563,7 +565,7 @@
         }
       },
       "SandboxRequestBody": {
-        "title": "Reembolsar cobrança",
+        "title": "Refund charge",
```
Comment (bcunhapicpay) [inline static/swagger/picpay-1-click-en.json:568]: Charge Refund

**Pull request merged**